src/main/java/frc/robot/subsystems/Hood.java

```diff
@@ -86,13 +97,21 @@ public double getSetpoint() {
    * @param speed the speed to set the servo to as a percent from -1 to 1
    */
   public void set(double speed) {
-    if ((getAngle() < Constants.Hood.MIN_ANGLE && speed < 0) || (getAngle() > Constants.Hood.MAX_ANGLE && speed > 0)) {
+    if ((getRawAngle() < Constants.Hood.MIN_RAW_ANGLE && speed < 0)
+        || (getRawAngle() > Constants.Hood.MAX_RAW_ANGLE && speed > 0)) {
```

Comment:
Use hood hard stops rather than pot hard stops

Reply:
resolved in acedb0d